pkg/util/process/https.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
is this specifically designed for kubelet or a common https client? for the former, you should move this file to `util/native/`; for the latter, you should avoid using kubelet-specified concepts in this file

Reply:
Considering it is currently designed for kubelet, I will move this file to "util/native".